transmon_pocket_teeth.py

```diff
@@ -0,0 +1,330 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+# Contributers: Figen YILMAZ, Li-Chieh Hsiao, Dr Christian K Andersen
+
+"""Transmon Pocket 6.

```

Comment:
New name here

Reply:
> Looks good to me.
> Did it render alright into Ansys?
F: Yes, that's why we are living for :) I just have an issue with the materials.